chain/default_syncer_test.go

```diff
@@ -922,3 +922,63 @@ func initGenesis(cst *hamt.CborIpldStore, bs bstore.Blockstore) (*types.Block, e
 		consensus.MinerActor(minerAddress, minerOwnerAddress, []byte{}, 1000, minerPeerID, types.ZeroAttoFIL),
 	)(cst, bs)
 }
+
+func TestIsReorg(t *testing.T) {
```

Comment:
whenever you see unit tests this complicated for a function that checks to see if a thing is in a list, something is wrong. you can use that as a signal to look for other ways, eg make the function simpler with fewer dependencies.